src/templates/finops-hub/modules/adxTableSchema.kql

```diff
@@ -94,3 +94,11 @@
   x_SkuTerm:string,
   x_SkuTier:string
   )
+
+.create-or-alter materialized-view FocusClean on table Focus
+{
+    Focus
+    | extend IngestionTime = ingestion_time()
+    | summarize arg_max(IngestionTime , *) by BillingAccountId, ChargePeriodStart, CommitmentDiscountId, Region, ResourceId, SkuPriceId, SubAccountId, x_AccountOwnerId, x_CostCenter, x_InvoiceSectionId, x_SkuDetails, x_SkuMeterId,
```

Comment:
Do we need `by BillingAccountId, ...`? We know those don't uniquely identify rows, so why are they used? Do we need every column or can we remove by "by" segment?

Reply:
No this is required, as that collection of columns uniquely identifies the meter/SKU/resource for the day it is charged (ChargePeriodStart). Arg_Max takes the latest entry (so on July 10, it will take the July 9 entry) based on those columns... since there is no way to uniquely identify a row. This is working incredibly well at the moment, based on 10 days of data so far.

Reply:
If that's the case, this won't work since there's no combination of columns that uniquely identify a row. Wouldn't this result in some records getting left out if they have the same values for each of those columns?

FYI - There are actually more columns needed to identify uniqueness (e.g., Tags), but even if you had all of them, that still won't be unique since there are multiple rows with these same values per day.

Reply:
I'll sign off on this, but we still need to revisit it as I don't think this is correct.